### ISSUE TITLE
Enhancement: Remove the 'You got a new response' title in contact form emails

### ISSUE THREAD

**Issue by johngodley** [Automattic/jetpack#31516]
### Impacted plugin

Jetpack

### What

Contact form emails contain an unnecessary title

### How

At some point recently the emails sent by the Jetpack contact form have a 'You got a new response' title added to the email body in a large font. As an informational bit of text this is not needed, but if you then reply to the message it gets included in the email and just looks very odd.

![image](https://github.com/Automattic/jetpack/assets/1277682/dbe6a140-d220-4ddb-ba8a-b3963b97bf36)

Each time I have to manually edit it out. Could we consider not having the title?

**Comment by jeherve** [comment]
Related: #30088, where this was added, alongside a filter that would allow you to remove this string if necessary:

```php
add_filter( 'jetpack_forms_response_email_title', '__return_empty_string' );
```

@donnchawp What would you think about removing the sentence by default, while keeping the filter so folks can add a title if they need it.

**Comment by jkipk** [comment]
Have been digging through Jetpack, Outlook, and M365 PowerAutomate looking for where this grammatically incorrect and  (grammar police alert!) personally offensive header was coming from (correct: "You have a new response" - incorrect: "You got a new response."
Adding the filter, but please remove it asap.

**Comment by upwardmomentum84** [comment]
This user would like the "You got a new response" to be removed from the emails if possible to they don't have to remove it every time they respond to an email: 6466359-zd-woothemes

**Comment by Oclair** [comment]
@jeherve What about the Admin urls added into the email as well?
<img width="175" alt="image" src="https://github.com/Automattic/jetpack/assets/619643/47259582-6847-44c4-9db6-5c1284b9dfe3">

Who is modifying thousands of websites on a whim anyways?
This contact form feature has been working for several years and now someone needs to change it's behaviour?

**Comment by jeherve** [comment]
> What about the Admin urls added into the email as well? 

@Oclair Thanks for the feedback. I opened #31774 to keep track of this.

I would also recommend that you update to the most recent version of the Jetpack plugin. This was fixed in the plugin a month ago in #31270, so once you update Jetpack those links will not appear in your emails anymore.
